### ISSUE TITLE
Gulp build generates wrong path (href and src)

### ISSUE THREAD

**Issue by lhas** [FountainJS/generator-fountain-webapp#155]
Hello!

I'm always using the FountainJS for bootstrapping my recent projects, since the Fountain only supported Angular 1. I just love this stack, man.

Today i started a new project, with "yo fountain-webapp".

I bootstrapped two components into it, and then i run `gulp build`.

After i tried to run the index.html generated by `build`, i get two errors on console:

```
vendor-6a7e6af81a.js.map:1 Uncaught SyntaxError: Unexpected token :
app-edb833c4ed.js.map:1 Uncaught SyntaxError: Unexpected token :
```

Then i look the source code of index.html from dist, and it looks like:

```
    <link rel="stylesheet" href="maps/styles/vendor-e197f9d7ed.css.map">
    <link rel="stylesheet" href="maps/styles/app-d41d8cd98f.css.map">
  <script src="maps/scripts/vendor-6a7e6af81a.js.map"></script>
  <script src="maps/scripts/app-edb833c4ed.js.map"></script>
```

And the correct is (in the old versions this work as expected, i'm really not sure what happened in fountain):

```
    <link rel="stylesheet" href="styles/vendor-e197f9d7ed.css">
    <link rel="stylesheet" href="styles/app-d41d8cd98f.css">
  <script src="scripts/vendor-6a7e6af81a.js"></script>
  <script src="scripts/app-edb833c4ed.js"></script>
```

Without the map references.

What is happening here? it disappointed me a lot to continue using the fountain :-1: 

Node.js v6.6.0  
linux 4.4.24-1-MANJARO
yo 1.8.5
npm 3.10.3

**Comment by lhas** [comment]
@MaicolBen this happens with you too? Fuck, i'm stuck with this.

**Comment by lhas** [comment]
I commented all references about sourcemaps i gulp_tasks/build.js and it worked as expected. But i want to keep using sourcemaps. What i need to do?

**Comment by MaicolBen** [comment]
I solved downgrading `gulp-sourcemaps` to 1.6.0
